lib/samples/filter_features_in_scene/README.md

```diff
@@ -0,0 +1,41 @@
+# Filter features in scene
+
+Filter 3D scene features out of a given geometry with a polygon filter.
+
+![Image of filter features in scene](filter_features_in_scene.png)
+
+## Use case
+
+You can directly control what users see within a specific scene view to give a more focused or cleaner user experience by using a `SceneLayerPolygonFilter` to selectively show or hide scene features within a given area.
+
+## How to use the sample
+
+The sample initializes showing the 3D buildings OpenStreetMap layer. Click the "Load detailed buildings" button to load an additional scene layer that contains more detailed buildings. Notice how the two scene layers overlap and clip into each other. Click the "Filter OSM buildings in extent" button, to set a `SceneLayerPolygonFilter` and filter out the OpenStreetMap buildings within the extent of the detailed buildings scene. Notice how the OSM buildings within and intersecting the extent of the detailed buildings layer are hidden. Click the "Reset scene" button to hide the detailed buildings scene layer and clear the OSM buildings filter.
```

Comment:
```suggestion
The sample initializes showing the 3D buildings OpenStreetMap layer. Tap the "Load detailed buildings" button to load an additional scene layer that contains more detailed buildings. Notice how the two scene layers overlap and clip into each other. Tap the "Filter OSM buildings in extent" button, to set a `SceneLayerPolygonFilter` and filter out the OpenStreetMap buildings within the extent of the detailed buildings scene. Notice how the OSM buildings within and intersecting the extent of the detailed buildings layer are hidden. Tap the "Reset scene" button to hide the detailed buildings scene layer and clear the OSM buildings filter.
```

Reply:
As well as addressing the click -> tap, these instructions don't match what's happening in the app. 

When the app first loads, I see a "Filter OSM Buildings" button. So we're missing an initial step here i.e. the buildings are already added currently. It looks like after resetting we go back to the start so just make sure the app starts at that reset position.

Then, review the steps here and ensure the calls to action on the buttons in the readme matches what's in the app. It's fine if there is a slight variance from the design - clearly this design is desktop since it references "click", so if there was a need for us to slightly shorten button calls to action for example, that is ok.